src/log-tlsstore.c

```diff
@@ -184,13 +242,22 @@ static void LogTlsLogPem(LogTlsStoreLogThread *aft, const Packet *p, SSLState *s
                 goto end_fwrite_fpmeta;
         }
 
+        subject = CreateStringFromByteArray(connp->cert0_subject, connp->cert0_subject_len);
+        issuerdn = CreateStringFromByteArray(connp->cert0_issuerdn, connp->cert0_issuerdn_len);
+
```

Comment:
Is this OK? `tls-log` is deprecated but `tls-store` is still there and logs certificate metadata as strings.